src/gr11review/part1/Review2.java

```diff
@@ -1 +1,45 @@
 package gr11review.part1;
+
+import java.io.*;
+
+public class Review2 {
+
+  public static void main(String[] args) throws IOException {
+    BufferedReader key = new BufferedReader(new InputStreamReader(System.in));
+
+    // Joke option outputs
+    System.out.println("0 - print a joke about your hair");
+    System.out.println("1 - print a joke about your feet");
+    System.out.println("2 - print a joke about your clothes");
+    System.out.println("3 - print a joke about your teacher");
+
+    // User input 
+    System.out.print("Choose a menu option: ");
+    int intOption = Integer.parseInt(key.readLine());
+
+    // Joke contents
+    switch (intOption) {
+      case 0 : 
+        System.out.println("Why do bees have sticky hair?");
+        System.out.println("They always use honeycombs.");
+        break;
+      case 1 : 
+        System.out.println("Did you hear about the four foot tall psychic who escaped prison?");
+        System.out.println("He's a small medium at large.");
+        break;
+      case 2 : 
+        System.out.println("What’s a baby’s favorite clothing brand?");
```

Comment:
Not going to take off marks for this, but this line 31 causes the autograding to fail because of the single quote characters in the `"What’s a baby’s favorite clothing brand?"`.  Sometimes if you copy text from a website this can cause an issue because of character formatting.